Change service creation flow

Loquacity opened the pull request:
# Description

Update the service creation flow to reflect the changes to the Allmilk demo.

# Links

Fixes https://github.com/timescale/docs-private/issues/52

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

Loquacity committed:
Change service creation flow

Loquacity commented:
**Install Cloud:**

![image](https://user-images.githubusercontent.com/3914967/190335003-602fa05d-eb6f-4763-a304-df63d0ce304c.png)

**Getting Started Guide:**

![image](https://user-images.githubusercontent.com/3914967/190335138-0e109780-9796-4b76-b246-4be9addd79b6.png)

lukas-bernert commented:
Suggestion for step 4:
the way you create a demo is now different. We don't have the tabs "with/without demo" anymore.

If users wants to create a demo, they need to click on the grey box underneath the price calculator that says: "Interactive demo. Deploy a service..."

Loquacity commented:
> Suggestion for step 4: the way you create a demo is now different. We don't have the tabs "with/without demo" anymore.
> 
> If users wants to create a demo, they need to click on the grey box underneath the price calculator that says: "Interactive demo. Deploy a service..."

Yep, that's why I changed the wording away from tabs, and to mirror the updated workflow. I also updated the image.

Loquacity committed:
Merge branch 'latest' into create-service-lana

Loquacity committed:
Merge branch 'latest' into create-service-lana

Loquacity committed:
Merge branch 'latest' into create-service-lana
